app/controllers/statuses_controller.rb

```diff
@@ -63,6 +63,7 @@ def update
     if @status.update(permitted_params.status)
       flash[:notice] = I18n.t(:notice_successful_update)
       redirect_to action: "index"
+      apply_status_p_complete_change
```

Comment:
```suggestion
      apply_status_p_complete_change
      redirect_to action: "index"
```

The `render` or `redirect` call should be the last thing in a controller action.

Reply:
You're totally right. I don't know why I did this.